recipes/noqt/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set version = "1.0" %}
+
+package:
+  name: noqt
+  version: {{ version }}
+
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - {{ pin_subpackage('noqt4', exact=True) }}
+    - {{ pin_subpackage('noqt5', exact=True) }}
+    - {{ pin_subpackage('noqt6', exact=True) }}
+test:
+  commands:
+    - echo "no tests for this package"
+
+outputs:
+  - name: noqt
+  - name: noqt4
+    build:
+      noarch: generic
+    requirements:
+      run_constrained:
+        # qt4 only ever had a "mega qt" package called qt
+        - qt <0.a0|>=5.a0
+    test:
+      commands:
+        - echo "no tests for this package"
+  - name: noqt5
+    build:
+      noarch: generic
+    requirements:
+      run_constrained:
+        # qt-main is essentially qt5
+        - qt-main <0.a0
+        # before the creation of qt-main, a megapackage called qt existed
+        - qt <0.a0
```

We need a minor version before the "a0". Also, I would only do `qt-main <5.0a0`, so we don't close the door that `qt6-main` eventually becomes `qt-main`. That the output got renamed was to enable co-installation; once qt5 fades away, we won't need that anymore. Not saying that's a good enough reason to then change it, but it might happen.
```suggestion
        # qt-main is essentially qt5
        - qt-main <5.0a0
        # before the creation of qt-main, a megapackage called qt existed
        - qt <0.0a0
```

---

Thanks o hope i addressed the 2 points.